spec/models/user_spec.rb

```diff
@@ -87,13 +89,53 @@
   end
 
   describe 'methods' do
+    describe '#attended_event?' do
+      context 'user has attended to the event' do
+        before { events_registration.update_attributes(attended: true) }
```

Comment:
@Ana06 Ok I think it's ready now. I'll make my best to stick to project conventions for the following commits 🙃